spyder/preferences/runconfig.py

```diff
@@ -88,6 +90,8 @@ def set(self, options):
         self.python_args_enabled = options.get('python_args/enabled', False)
         self.clear_namespace = options.get('clear_namespace',
                                     CONF.get('run', 'clear_namespace', False))
+        self.console_namespace = options.get('console_namespace',
+                                   CONF.get('run', 'console_namespace', False))

```

Comment:
This a PEP error but fixing it doesn't make it more readable

Reply:
Ok, don't worry about it.